homeassistant/components/template/strings.json

```
@@ -324,24 +402,37 @@
           "device_id": "[%key:common::config_flow::data::device%]",
           "name": "[%key:common::config_flow::data::name%]",
           "state": "[%key:component::template::common::state%]",
-          "start": "Actions on turn off",
+          "start": "Actions on start",
           "fan_speed": "Fan speed",
           "fan_speeds": "Fan speeds",
           "set_fan_speed": "Actions on set fan speed",
           "stop": "Actions on stop",
           "pause": "Actions on pause",
-          "return_to_base": "Actions on return to base",
+          "return_to_base": "Actions on return to dock",
           "clean_spot": "Actions on clean spot",
           "locate": "Actions on locate"
         },
         "data_description": {
-          "device_id": "[%key:component::template::common::device_id_description%]"
+          "device_id": "[%key:component::template::common::device_id_description%]",
+          "state": "Defines a template to get the state of the vacuum. Valid values are `cleaning`, `docked`, `idle`, `paused`, `returning`, and `error`.",
+          "start": "Defines actions to run when the vacuum is started.",
+          "fan_speed": "Defines a template to get the fan speed of the vacuum.",
+          "fan_speeds": "List of fan speeds supported by the vacuum.",
+          "set_fan_speed": "Defines actions to run when the vacuum is given a command to set the fan speed. Receives variable `fan_speed`",
+          "stop": "Defines actions to run when the vacuum is stopped.",
+          "pause": "Defines actions to run when the vacuum is paused.",
+          "return_to_base": "Defines actions to run when the vacuum is given the return to dock command.",
+          "clean_spot": "Defines actions to run when the vacuum is given a clean spot command.",
+          "locate": "Defines actions to run when the vacuum is given a locate command."
```

As we refer to (friendly) command names we should quote them:

```suggestion
          "return_to_base": "Defines actions to run when the vacuum is given the 'Return to dock' command.",
          "clean_spot": "Defines actions to run when the vacuum is given a 'Clean spot' command.",
          "locate": "Defines actions to run when the vacuum is given a 'Locate' command."
```